src/hooks/useDarkMagic.ts

```diff
@@ -61,3 +63,21 @@ const cargo = async (arg0: string | undefined) => {
 
   return undefined
 }
+
+const brew = async (arg0: string | undefined) => {
+  if (!Path.findBin("brew")) return undefined
+
+  const res = await useFetch(`https://formulae.brew.sh/api/formula/${arg0}.json`)
```

Comment:
this probably works a reasonable amount of the time. I would however prefer us to run a service that installs all formula and indexes their bin dirs as a longer term project tho.